config/settings.yml

```diff
@@ -29,3 +29,10 @@ plugins:
   patreon_login_ignore_state:
     default: true
     hidden: true
+  patreon_donation_prompt_enabled:
+    default: true
```

Comment:
Can't have this a default true.

Reply:
Updated: https://github.com/discourse/discourse-patreon/pull/29/commits/4ba62212339f0eabeaea04c3d5b3469c2812ea57